docs/rules/useSelector-prefer-selectors.md

```diff
@@ -0,0 +1,59 @@
+#  Enforces that all useSelector properties use selector functions. (react-redux/useSelector-prefer-selectors)
+
+Using selectors in `useSelector` to pull data from the store or [compute derived data](https://redux.js.org/recipes/computing-derived-data#composing-selectors) allows you to uncouple your containers from the state architecture and more easily enable memoization. This rule will ensure that every prop utilizes a selector.
```

Comment:
uncouple --> decouple

(sorry for being picky :) )

Reply:
again, I think that anonymous functions are technically selectors as well. The rule enforces that these selectors are defined separately as apposed inline. Would be nice to clarify that, if that is correct.